src/main/java/flight/reservation/plane/Helicopter.java

```diff
@@ -1,25 +1,19 @@
 package flight.reservation.plane;
 
-public class Helicopter {
-    private final String model;
-    private final int passengerCapacity;
+public class Helicopter extends AirCraft {
 
     public Helicopter(String model) {
         this.model = model;
         if (model.equals("H1")) {
-            passengerCapacity = 4;
+            this.passengerCapacity = 4;
         } else if (model.equals("H2")) {
-            passengerCapacity = 6;
+            this.passengerCapacity = 6;
         } else {
             throw new IllegalArgumentException(String.format("Model type '%s' is not recognized", model));
         }
-    }
+        this.crewCapacity = 0;
```

Comment:
perhaps you can refactor this to use a base class constructor which gets passed the parameters, as model, passengerCapacity, and crewCapacity are members of the base class